blog/2015/12/call-for-candidates-elections-2016.html

```diff
@@ -0,0 +1,55 @@
+---
+layout: blog
+root: ../../..
+author: Jonah Duckles
+title: "Call for Candidates for the 2016 Steering Committee"
+date: 2015-12-08
+time: "06:00:00"
+category: ["Community", "Elections", "Steering Committee"]
+---
+
+<!-- start excerpt -->
+<p>From February 15-19, Software Carpentry will hold its annual election for the
+Steering Committee of The Software Carpentry Foundation. The inaugural Steering
+Committee has been an incredible team of dedicated community members working to
+position Software Carpentry so that it has a strong foundation and continued
+impact in scientific communities around the world. Please consider giving
+back to the community by standing for election!</p>
+<!-- end excerpt -->
+
+<p>If you are a qualified instructor who has taught at least twice in the past
+two years, or have done non-teaching work for the community, you can both stand
+for election and vote.</p>
+
+<p>In order to stand for election we request that
+you write a blog post that introduces yourself to the community. The post:</p>
+<ul>
+  <li>must be about 500 words </li>
+  <li>can be written in any format (question & answer, paragraph etc.),
+and</li>
+  <li> must be titled &quot;2016 Election: Your Name&quot;</li>
+</ul>
+
+<p>You can submit your post as a pull requestion to <a
+href="https://github.com/swcarpentry/site">this repository</a> or by email <a
+href="mailto:admin@software-carpentry.org">admin@software-carpentry.org</a>. In
+the post, you should explain:</p>
+<ul>
+  <li>your background how you've been involved in Software Carpentry</li>
+  <li>what you will do as a member of the Steering Committee to contribute to the growth and success of the community</li>
+</ul>
+
+<p>Blog posts must be submitted by Friday February 5th, one
+full week befor the election.</p>
```

Comment:
Ok, moving the post to state 30-days for blog posts. I was going off Greg's initial post about elections where he said 7-days. But I'm not sure bylaws were in place at that time. That gets us blog posts due by Jan 15th.